BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/BottomNavigation/SkontoNavigationBarBottomAdapter.swift

```diff
@@ -48,4 +48,18 @@ public protocol SkontoNavigationBarBottomAdapter: InjectedViewAdapter {
      * - Parameter enabled: A bool value to reflect the state of the badge
      */
     func updateDiscountBadge(enabled: Bool)
+
+    /**
+     *  Set the saved amount text on the bottom navigation bar. Called when the saved amount needs to be displayed.
+     *
+     * - Parameter text: A string which contains the value of the saved amount.
+     */
+    func updateSavedAmount(with text: String?)
```

Comment:
[Suggestion] Maybe a better name like"updateInvoiceSkontoSavings" and "A string that contains the value of the saved amount when paying within the Skonto period.". Or something like this.
As well for this:
"Set the saved amount text on the bottom navigation bar" -> "Set the saved amount text on the bottom navigation bar. This reflects the savings after paying the invoice within the Skonto period."